src/bundle/types.ts

```diff
@@ -98,6 +98,10 @@ export type MarkdownEditorMarkupConfig = {
     languageData?: YfmLangOptions['languageData'];
     /** Config for @codemirror/autocomplete https://codemirror.net/docs/ref/#autocomplete.autocompletion%5Econfig */
     autocompletion?: CreateCodemirrorParams['autocompletion'];
+    /**
+     * The function, used to determine if the pasted text is the image url and should be inserted as an image
+     */
+    parseInsertedUrlAsImage?: (text: string) => {imageUrl: string; title?: string} | null;
```

Comment:
nit:
Lets create type for `(text: string) => {imageUrl: string; title?: string} | null`
and we will use it in WYSIWYG and markup